CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 ## Upgrade Notes
 
+### Changed
```

Comment:
We're taking care of the CHANGELOG.md, please, don't modify anything in it unless the PR requires some "Upgrade notes" for the community